### PR TITLE
added release and zulip badget

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Mixxx 
+[![GitHub release](https://img.shields.io/github/release/mixxxdj/mixxx.svg)](https://github.com/mixxxdj/mixxx/releases/latest)
```
Comment (uklotzde) [inline README.md]: Please use https://img.shields.io/github/tag/mixxxdj/mixxx.svg = _GitHub tag (latest SemVer)_ for selecting the most recent version. Selecting releases by date is confusing and may alternate between different release branches. Currently it is showing 2.1.6.

**Review by uklotzde — CHANGES_REQUESTED** [review]
Wrong release badge

**Comment by daschuer** [comment]
OK works now. Thank you for the hint. 
I have not managed to link to the tag, so now it links to our downloads page.
This is IMHO more reasonable anyway.

**Comment by daschuer** [comment]
Yippie, I have got #2000 :tada:

**Comment by rryan** [comment]
> Yippie, I have got #2000 🎉

And you had #1000 too. Lucky!

**Comment by rryan** [comment]
> @Be-ing has the honor of #1000

Doh, I mis-read the top-line. You merged it.

**Review by uklotzde — APPROVED** [review]

**Pull request merged**